tyk-vcs/getter.go

```diff
@@ -222,23 +227,32 @@ func fetchAPIDefinitionsDirect(fs billy.Filesystem, spec *TykSourceSpec, subdire
 			ad.APIDefinition = &def
 		}
 
-		if defInfo.APIID != "" {
-			ad.APIID = defInfo.APIID
-		}
+		if ad.OAS != nil {
+			oasAPICount++
+			ad.APIDefinition = &objects.APIDefinition{APIDefinition: apidef.APIDefinition{IsOAS: true}}
+		} else {
+			classicAPICount++
 
-		if defInfo.DBID != "" {
-			ad.Id = model.ObjectIDHex(defInfo.DBID)
-		}
+			if apiFileInfo.APIID != "" {
```

Comment:
Actually, APIID, DBID, and ORGID are not parsed from the spec file (.tyk.json). As I am not sure why this logic is added to classic APIs, I didn't delete it from classic API logic. But for both types of APIs (Classic, OAS), APIID, DBID, ORGID, and OAS fields in tyk-spec are empty as `.tyk.json` file does not contain these fields in it

Reply:
Actually, you can provide it in `.tyk.json` file.
If it doesn't work even after providing it, you can ignore this.

Reply:
i just updated codebase as you suggested just in case; however, i couldn't set explicit org_id through .tyk.json file even for classic and OAS APIs. It may not be supported. worth to investigate in another ticket.